### DISCUSSION TITLE
✅ Swiper Carousel Loop Beta v3.11.0 - Centered slides

### DISCUSSION THREAD

**Discussion by itaides** [elementor/elementor#21195] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Its will be great if we can set in the new loop carousel option to set each number slides we want for example: 2.5 slides per view. centered slides and set Keyboard/Mousewheel control. 

### Describe the solution you'd like

1.Adding Option to set centered slides (true or false).
2.Adding Option to set Keyboard control (true or false).
3.Adding Option to set Mousewheel control (true or false).
4. Adding Option to set Each number we want in Slides to display.
5. Adding Option to set custom query like in posts widget ( for using acf field relationship)

**Comment by ayoubkhan558** [comment]
Also add auto height swiper

**Comment by vladninja** [comment]
Duplicate: https://github.com/orgs/elementor/discussions/21453 
But also want to be able to use center mode

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
